### PR TITLE
Re-export useSignals in @preact/signals-react

### LINKED ISSUE #473: @preact/signals-react@2.0.0 not re-exporting useSignals
According [to the docs](https://github.com/preactjs/signals/blob/d7f43ef5c9b6516cd93a12c3f647409cfd8c62be/packages/react/README.md#usesignals-hook), I should be able to `import { useSignals } from '@preact/signals-react'`, but since `useSignals` is not re-exported, I need to do this instead: `import { useSignals } from '@preact/signals-react/runtime'`.

### PR DESCRIPTION
**Pull request by JonAbrams**
Fixes #473

### CODE REVIEW

File: .changeset/wild-deers-leave.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@preact/signals-react": minor
```
Comment (andrewiggins) [inline .changeset/wild-deers-leave.md:2]: Let's make this a `patch` change since this a bug fix with no new feature introductions

**Review by andrewiggins — CHANGES_REQUESTED** [review]
Thanks! Let's adjust the changeset release and get this out :)

**Comment by andrewiggins** [comment]
Actually, just saw that the Readme was updated. Let's roll with that for now. Thanks for opening this though.

**Comment by rschristian** [comment]
@andrewiggins `react-transform` doesn't re-export core though, do we want this?

```js
import { signal, effect } from '@preact/signals-react';
import { useSignals, useSignal } from '@preact/signals-react-runtime';
```

I'm not super in love with this myself.

**Pull request closed without merging**